src/pages/APIPage.tsx

```diff
@@ -90,13 +90,13 @@ export const APIPage: FunctionComponent = () => {
             <SidebarPanel className="pf-u-p-lg">
                 <SidebarApiSections openapi={openapi} groupedOperations={groupedOperations} />
             </SidebarPanel>
-
             <SidebarContent>
               { (apiState.isLoading || !apiState.api || groupedOperations.loading) ?
                   <Bullseye><Spinner /></Bullseye> :
                   <ApiDoc apiContent={apiState.api} groupedOperations={groupedOperations.value} /> }
             </SidebarContent>
           </Sidebar>
+          <BackToTop  style={{position: 'fixed'}}/>
```

Comment:
could you move the style to [App.scss](https://github.com/RedHatInsights/api-documentation-frontend/blob/main/src/App.scss#L78) so that all our styles are in a single file?

It would be inside `.apid-c-page-apipage`  and something like this:
```
.pf-c-back-to-top {
    position: fixed;
  }
```

Reply:
Added to app.scss thanks guys!